applications/models/base.py

```diff
@@ -52,7 +52,7 @@ class Meta:
 
     # ABOUT YOU
     # Population analysis, optional
-    gender = models.CharField(max_length=23, choices=GENDERS, default=NO_ANSWER)
+    gender = models.CharField(max_length=23, choices=GENDERS, default=NO_ANSWER, blank=True, null=True)
```

Comment:
No cal perquè per defecte es no answer, aleshores mai serà null, sinó que si no seleccionen res serà prefer not to answer